src/makeCalculator.js

```diff
@@ -4,7 +4,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(number) {
+      this.result -= number;
+    },
+
+    multiply(number) {
+      this.result *= number;
+    },
+
+    divide(number) {
+      if (number === 0) {
+        console.log('Cannot divide by zero');
+        return;
+      }
```

Comment:
The `divide` method currently logs a message to the console when trying to divide by zero. It would be better to throw an error instead, as logging to the console is not a reliable way to handle errors in a production environment.